xrpl4j-core/src/main/java/org/xrpl/xrpl4j/crypto/keys/bc/BcKeyUtils.java

```diff
@@ -100,8 +100,10 @@ public static PrivateKey toPrivateKey(final Ed25519PrivateKeyParameters ed25519P
   public static PrivateKey toPrivateKey(final ECPrivateKeyParameters ecPrivateKeyParameters) {
     // Convert the HEX representation of the BigInteger into bytes.
     byte[] privateKeyBytes = BaseEncoding.base16().decode(ecPrivateKeyParameters.getD().toString(16).toUpperCase());
-    final UnsignedByteArray privateKeyUba = UnsignedByteArray.of(privateKeyBytes);
-    return PrivateKey.of(privateKeyUba);
+    return PrivateKey.of(
```

Comment:
Just to double check -- We don't need to do any prefix stripping in `toEcPrivateKeyParams`?

Reply:
This has been fixed. I'm now padding all public and private key byte arrays to ensure they're always appropriately sized (regardless of what BigInteger's conversion operations yield to us).